header-footer-grid/Core/Builder/Abstract_Builder.php

```diff
@@ -1072,16 +1073,17 @@ private function add_row_style( $row_index, $css_array = array() ) {
 					'--bgImage'          => [
 						Dynamic_Selector::META_KEY    => $this->control_id . '_' . $row_index . '_background',
 						Dynamic_Selector::META_FILTER => function ( $css_prop, $value, $meta, $device ) {
-							$image = 'none';
+							$image     = 'none';
+							$image_url = apply_filters( 'nv_header_booster_featured_image_url', $value['imageUrl'], $value['useFeatured'] );
```

Comment:
we can apply the filter on the final value, i.e image, and use this filter key.
`nv_header_row_featured_image_url` and sending $image, $use_featured, $meta to allow full customization by the final image by any context if people want to tweak the bg image. 

Right now if we are not on single, they can't change the final value.

Also, please document the filter according, for reference on what standard you should use please search in code other documented filters.